src/lwc/geFormField/geFormField.js

```diff
@@ -46,12 +46,30 @@ export default class GeFormField extends LightningElement {
     }
 
     connectedCallback() {
-        const { defaultValue } = this.element;
-        if(defaultValue) {
+        const { defaultValue, recordValue } = this.element;
```

Comment:
I forgot that destructuring does not work when `this.element.defaultValue` or `this.element.recordValue` is `undefined`. We should change it to be something along the lines of:

```
const defaultValue = this.element.defaultValue;
const recordValue = this.element.recordValue;
```